willuslib/willus.h

```diff
@@ -195,7 +195,15 @@ typedef double  real;
 #if (defined(__linux) || defined(linux) || defined(__linux__))
 #define LINUX
 #if (defined(WILLUS_HAVE_FILE64) && !defined(_off64_t))
+#ifdef __off64_t
 #define _off64_t __off64_t
+#else
+/*
+** No, __off64_t exists, since we set _FILE_OFFSET_BITS earlier we can
```

Comment:
I imagine it doesn't matter, but you seem to be setting it later. ;-)

Reply:
Yeah... this is why I thought it probably didn't matter. Will remove `_FILE_OFFSET_BITS` if there are no further comments.

Reply:
Yeah, I'd get rig of the `_FILE_OFFSET_BITS`, and link to https://wiki.musl-libc.org/faq.html#Q:-Do-I-need-to-define-%3Ccode%3E_LARGEFILE64_SOURCE%3C/code%3E-to-get-64bit-%3Ccode%3Eoff_t%3C/code%3E? somewhere to explain the rationale.